templates/components/lists/objectives_list.html

```diff
@@ -143,8 +161,35 @@ <h4 slot="header">[[modalHeader]]</h4>
       </tr>
     </tbody>
   </table>
+  <!-- Use text in props -->
+
+  <!-- <b-table
+      id="objectivesTable"
+      :items="filtered_objectives"
+      :per-page="perPage"
+      :current-page="currentPage"
+      
+    ></b-table> -->
+
+  <b-pagination
+    v-model="currentPage"
+    :total-rows="rows"
+    :per-page="perPage"
+    prev-text="Prev"
+    next-text="Next"
+    class="mt-4"
+    aria-controls="objectivesTable"
+  >
+    <!-- <template v-slot:prev-text><span class="text-danger">Prev</span></template>
```

Comment:
@ninetteadhikari is there a reson why this commented block is left? May be you just delete it if it won't be used

Reply:
ok thanks